projects/android/gradlew

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env sh
+
+##############################################################################
+##
+##  Gradle start up script for UN*X
+##
+##############################################################################
+
+# Attempt to set APP_HOME
+# Resolve links: $0 may be a link
+PRG="$0"
+# Need this for relative symlinks.
+while [ -h "$PRG" ] ; do
+    ls=`ls -ld "$PRG"`
```

Comment:
nit: modern style is to use $()
```
ls=$(ls -ld "$PRG")
```

Reply:
Hmm, were these scripts written from scratch or based on some existing ones?

Reply:
Doesn't look like just a style issue, apparently backtick is more error-prone:
http://mywiki.wooledge.org/BashFAQ/082

Reply:
Also `gradlew` is generated by Graddle.

Reply:
That's correct, this script is auto-generated by Gradle.

Reply:
OK, so no need to review this file then! BTW I highly recommend https://www.shellcheck.net/ for checking shell scripts. It's available as a standalone program also.